src/PhpSpreadsheet/Reader/Gnumeric/PageSetup.php

```diff
@@ -68,7 +63,7 @@ public function sheetMargins(SimpleXMLElement $sheet): self
 
     private function buildMarginSet(SimpleXMLElement $sheet, array $marginSet): array
     {
-        foreach ($sheet->PrintInformation->Margins->children($this->gnm, true) as $key => $margin) {
+        foreach ($sheet->PrintInformation->Margins->children(Gnumeric::NAMESPACE_GNM) as $key => $margin) {
```

Comment:
Do we have any examples of the older `gmr` namespaced files that would allow us to test with them as well?

Reply:
Yes, I added a very small old.gnumeric with gmr namespacing to samples/templates a while back. It has been used ever since in tests in IOFactoryTest and GnumericLoadTest.
I will push the changes you requested tonight.
FYI, Xlsx is proving to be far more difficult than Gnumeric. I don't expect to have anything ready for weeks.